client/src/components/formHelpers/FormFields.js

```diff
@@ -49,14 +49,12 @@ function FormFields(props) {
           const type = name in types ? types[name] : 'text';
           return (
             <div className='inline-form-field' key={key}>
-              <Col sm={3} xs={12}>
+              <Col xs={12}>
```

Comment:
Let's do this
```diff
- <div className='inline-form-field' key={key}>
-   <Col xs={12}>
+ <Col xs={12}>
+   <FormGroup key={key}>
```
and remove css definitions for `inline-form-field`

Reply:
I wrapped it in
 ``` 
<Col key={key} xs={12}>
    <FormGroup>
```
as you requested, but removing the styles would cause the input to stick to the button. so I gave the styles to the form group instead.

Reply:
yeah, on another try, I see your point now. 👏 👏 👏